### PR TITLE
Add a cleaner daemon

### PR DESCRIPTION
**Pull request by PouuleT**
The cleaner deletes the torrent when the ratio is reached
The cleaner deletes all the non-needed files

### CODE REVIEW

File: server/cleaner.go [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+	"sync"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/odwrtw/polochon/lib"
+)
+
+type cleaner struct {
+	config     *Config
+	videoStore *polochon.VideoStore
```
Comment (nduhamel) [inline server/cleaner.go]: videoStore not used

File: server/cleaner.go [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+	deletePath := filepath.Join(c.config.Downloader.Cleaner.TrashDir, path.Base(directoryPath))
+
+	log.Debug("Deleting folder ... ")
+	log.Debugf("%s ---> %s", directoryPath, deletePath)
+
+	return os.Rename(directoryPath, deletePath)
+}
+
+// Run launches the cleaner
+func (app *App) startCleaner() {
```
Comment (nduhamel) [inline server/cleaner.go]: Maybe this function have to be in app.go

Comment (PouuleT) [inline server/cleaner.go]: Absolutely right!

File: server/cleaner.go (hunk 1 above)
Comment (PouuleT) [inline server/cleaner.go]: Deleted!

**Pull request merged**